src/jsifier.js

```diff
@@ -114,6 +114,16 @@ function JSify(data, functionsOnly) {
 
   // functionStub
   function functionStubHandler(item) {
+    if (!DISABLE_EXCEPTION_THROWING) {
+    // In LLVM, exceptions generate a set of functions of form __cxa_find_matching_catch_1(), __cxa_find_matching_catch_2(), etc.
+    // where the number specifies the number of arguments. In Emscripten, route all these to a single function '__cxa_find_matching_catch'
+    // that variadically processes all of these functions using JS 'arguments' object.
+      if (item.ident.startsWith('___cxa_find_matching_catch_')) {
+        var num = +item.ident.split('_').slice(-1)[0];
+        addCxaCatch(num);
```

Comment:
Can we add a return here?

Also can you combine the conditions?:

````
if (!DISABLE_EXCEPTION_THROWING && item.ident.startsWith('___cxa_find_matching_catch_') {
```

Also what happens if we have DISABLE_EXCEPTION_THROWING set but then the module wants one of these functions.. I wonder if we should error here:

```
if (item.ident.startsWith('___cxa_find_matching_catch_')) {
   if (DISABLE_EXCEPTION_THROWING)
      some_kind_of_error?
   ...
   return early;
}
```

That last part is optional :)

Reply:
Good idea, adding an error.

We can't early return here because we just add library code, but emitting it is handled lower down. I added a comment.